package.json

```diff
@@ -33,7 +33,7 @@
     "@shelex/cypress-allure-plugin": "^2.23.0",
     "babel-eslint": "^10.1.0",
     "bigtest": "^0.14.0",
-    "cypress": "^10.5.0",
+    "cypress": "^9.1.0",
```

Comment:
"cypress": "^9.1.1",
was before. Please update the version

Reply:
done